AppCenterCrashes/AppCenterCrashes/Internals/MSCrashesInternal.h

```diff
@@ -61,17 +61,29 @@ static NSString *const kMSAppDidReceiveMemoryWarningKey = @"MSAppDidReceiveMemor
  */
 - (void)configureCrashReporterWithUncaughtExceptionHandlerEnabled:(BOOL)enableUncaughtExceptionHandler;
 
-/*
+/**
  * Track handled exception directly as model form.
- * This API is not public and is used by wrapper SDKs.
+ * This API is used by wrapper SDKs.
  *
  * @param exception model form exception.
  * @param properties dictionary of properties.
  * @param attachments a list of attachments.
+ *
+ * @return handled error ID.
+ */
+- (NSString *)trackModelException:(MSException *)exception
+                   withProperties:(nullable NSDictionary<NSString *, NSString *> *)properties
+                  withAttachments:(nullable NSArray<MSErrorAttachmentLog *> *)attachments;
+
+/**
+ * Get a generic error report representation for an handled exception.
+ * This API is used by wrapper SDKs.
+ *
+ * @param errorId handled error ID.
+ *
+ * @return an error report.
  */
-+ (void)trackModelException:(MSException *)exception
-             withProperties:(nullable NSDictionary<NSString *, NSString *> *)properties
-            withAttachments:(nullable NSArray<MSErrorAttachmentLog *> *)attachments;
+- (MSErrorReport *)buildHandledErrorReportWithId:(NSString *)errorId;
```

Comment:
This new method should be moved to MSWrapperCrashesHelper like intended in the task.

Reply:
The method is already declared in `MSWrapperCrashesHelper.h` (https://github.com/microsoft/appcenter-sdk-apple/pull/1853/files#diff-42730e5ec8755042346f7627ac4850d9R86).

It was added to `MSCrashesInternal.h` because MSWrapperCrashesHelper needs to access this non-public API hence it has to be declared in private header file.